pkgs/top-level/all-packages.nix

```diff
@@ -26814,4 +26814,8 @@ in
   vpsfree-client = callPackage ../tools/virtualization/vpsfree-client {};
 
   gpio-utils = callPackage ../os-specific/linux/kernel/gpio-utils.nix { };
+
+  html5validator = callPackage ../applications/misc/html5validator {
+    inherit (python36Packages) buildPythonPackage nose;
+  };
```

Comment:
please attempt to sort this as well

Reply:
I moved this to the first plausible position